plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/util/Validator.js

```diff
@@ -33,8 +34,42 @@ Ext.define('NX.util.Validator', {
   },
 
   /**
+   * @private
+   */
+  nxNameRegex : /^[^(_)#A-Z\\\/"'\*\?<>|\s,.][^#A-Z\\\/"'\*\?<>|\s,.]*$/,
```

Comment:
This should cover both valid characters for elastic search index, and URL. Please feel free to suggest better regex to accomplish the same task, and any other constraints we might like to enforce for names in our domain :)